### PR TITLE
Adding new loading process for Solo Widget

### PR DESCRIPTION
**Pull request by rfornes**

### CODE REVIEW

File: layouts/solo/loader.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        };
+
+        //Dynamically creates an iframe & appends it's required CSS & JS libraries.
+        var getIframeHtml = function(options) {
+            var html = '<head><base target="_blank" /></head><body class="' + (options.className || '') + '" data-domain="' +
+                (options.domain || '') + '" data-id="' + (options.id || '') + '" onload="' +
+                'var d = document, head = d.getElementsByTagName(\'head\')[0],' +
+                'addJS = function(u){ var s = d.createElement(\'script\');s.src=u;d.body.appendChild(s);},' +
+                'addCSS = function(h){ var l = d.createElement(\'link\');l.type=\'text/css\';l.rel=\'stylesheet\';l.href=h;head.appendChild(l);};' +
+                'window.DEBUG=' + DEBUG + ';';
```
Comment (liorkuyer) [inline layouts/solo/loader.html]: TVP_DEBUG?

File: layouts/solo/loader.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        __TVPage__.config = __TVPage__.config || {};
+        __TVPage__.config["{{ $id }}"] = extend({{ $.Scratch.Get "settings" }}, __TVPage__.config["{{ $id }}"]);
+
+        var holder = document.getElementById("{{ $id }}-holder");
+        var iframe = holder.querySelector(".tvp-iframe");
+        var config = __TVPage__.config["{{ $id }}"];
+        var staticPath = "{{ .Type | absURL }}";
+
+        var dataMethod = "static";
+        if (isset(config, "channel") && isset(config.channel, "id")) {
```
Comment (liorkuyer) [inline layouts/solo/loader.html]: Checking the config is safer than channel type (entityType)?

File: static/solo/js/libs/player.js [hunk 3]
```diff
@@ -140,14 +129,14 @@
         config.logUrl = asset.analyticsLogUrl;
         analytics.initConfig(config);
       } else {
-        config.logUrl = '\/\/api.tvpage.com\/v1\/__tvpa.gif';
+        config.logUrl = '//api.tvpage.com/v1/__tvpa.gif';
```
Comment (liorkuyer) [inline static/solo/js/libs/player.js]: move to config?

**Comment by rfornes** [comment]
@liorkuyer applying feedback

File: layouts/solo/loader.html (hunk 2 above)
Comment (rfornes) [inline layouts/solo/loader.html]: Yes, I'm considering that a widget uses a "dynamic data method" as long as a channel object (standard or programmatic) is being passed. If a widget has to fetch its data and build itself on the fly, then it fits in this category.

Please let me know if you consider this is robust.

File: layouts/solo/loader.html (hunk 1 above)
Comment (rfornes) [inline layouts/solo/loader.html]: Yes, updating this.

File: static/solo/js/libs/player.js (hunk 3 above)
Comment (rfornes) [inline static/solo/js/libs/player.js]: Yes, will do.

**Pull request closed without merging**